oughta/oughta.cabal

```diff
@@ -1,26 +1,26 @@
 cabal-version: 2.4
-name: filecheck
+name: oughta
 version: 0.1.0.0
 author: Galois, Inc.
 maintainer: grease@galois.com
 copyright: Galois, Inc. 2025
 license: BSD-3-Clause
 license-file: LICENSE
-synopsis: Like LLVM's FileCheck tool, but as a Haskell library.
+synopsis: A tool to test programs that output text.
 description:
-  Like LLVM's FileCheck tool, but as a Haskell library.
+   A tool to test programs that output text.
```

Comment:
While we're at it, should we change "tool" to "library" here? As I understand it, Oughta isn't a standalone tool like LLVM's FileCheck.